src/components/molecules/PopapPokemons/PopapPokemons.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFetch } from "../../../hooks/useFetch";
+import { ReturnResult } from "../../../models/Pokemons";
+import { ResponsePokemon } from "../../../models/ResponsePokemon";
+import style from "../PopapPokemons/PopapPokemons.module.scss";
+
+interface Props {
+  pokemon: ReturnResult;
+  setShowPopup: () => void;
+}
+
+const urlDefault = 'https://pokeapi.co/api/v2/pokemon/'
```

Comment:
If that URL is used in different components, let's move it into a constant generic file, or an util